Makefile

```diff
@@ -39,6 +39,8 @@ CURRENT_TAG := $(shell ./version.sh)
 GRAVITY_TAG := $(CURRENT_TAG)
 # Abbreviated gravity version to use as a build ID
 GRAVITY_VERSION := $(CURRENT_TAG)
+# Release of the gravity runtime application to builder installer with intermediate steps
+GRAVITY_INTERMEDIATE_RELEASE ?= 6.1.31
```

Comment:
Will we want to keep this intermediate up to date with the latest 6.1 release? If so we should add an extra comment on when to update this value.

Reply:
This is a good question. That entry is mainly for the internal consumption when we build the installer tarball. The version choice is also sort of arbitrary - I went with the latest LTS release at the time I started working on it but there's no compelling reason to always use the latest.
I'll try to formulate a requirement for this as a comment.